packages/core/src/shared/crypto.ts

```diff
@@ -30,3 +30,15 @@ export function randomUUID(): `${string}-${string}-${string}-${string}-${string}
 
     return require('crypto').randomUUID()
 }
+
+/**
+ * Returns true if the given string is a UUID
+ *
+ * NOTE: There are different UUID versions, this function does not discriminate between them.
+ * See: https://stackoverflow.com/questions/7905929/how-to-test-valid-uuid-guid
+ */
+export function isUUID(uuid: string): boolean {
+    // pattern match the uuid to a uuid string
```

Comment:
looks like a normal regex, what is the purpose of this comment? 

however it's worth mentioning:

    // Must match "SessionID" pattern in telemetry/service-2.json

Reply:
A stale comment I used to get Q to generate a pattern. Deleting this, but will add the new comment you suggested.